src/modules/forum/actions.ts

```diff
@@ -0,0 +1,23 @@
+import { action } from 'typesafe-actions'
+import { Collection } from 'modules/collection/types'
+import { ForumPost } from './types'
+
+// Create collection forum post
+
+export const CREATE_COLLECTION_FORUM_POST_REQUEST = '[Request] Create collection forum post'
+export const CREATE_COLLECTION_FORUM_POST_SUCCESS = '[Success] Create collection forum post'
+export const CREATE_COLLECTION_FORUM_POST_FAILURE = '[Failure] Create collection forum post'
+
+export const createCollectionForumPostRequest = (collection: Collection, forumPost: ForumPost) =>
+  action(CREATE_COLLECTION_FORUM_POST_REQUEST, { collection, forumPost })
+export const createCollectionForumPostSuccess = (collection: Collection, forumLink: string) =>
+  action(CREATE_COLLECTION_FORUM_POST_SUCCESS, {
+    collection,
+    forumLink
+  })
+export const createCollectionForumPostFailure = (collection: Collection, error: string) =>
```

Comment:
Do you think that we need to use this action in the reducer to display an error or something? If the post to the forum fails, there is no way to retry.  Maybe we can add an issue to cover retries of this kind later to solve weird states.

Reply:
I think creating an issue is a good idea here because this is an action hidden to the user, so it's not like showing and error on the plubished collection makes sense.

I've created this issue to track it: https://github.com/decentraland/builder/issues/1290